src/components/elements/Icon.astro

```diff
@@ -0,0 +1,31 @@
+---
+interface Props {
+  svgPaths: string[];
+  color: string;
+  width: string;
+  height: string;
+  bgColor: string;
+  rounded: string;
+}
+
+const { svgPaths, color, width, height, bgColor, rounded } = Astro.props;
+---
+
+<div class={`rounded-${rounded} ${bgColor} p-2 w-10 h-10`}>
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={width}
+    height={height}
+    viewBox="0 0 24 24"
+    fill="none">
+    {svgPaths.map((path) => (
```

Comment:
@CodeLeom, help me understand why we have to loop through an SVG path here.

Are we doing something special with the SVGs?